src/components/Kv/KvHero.vue

```diff
@@ -10,15 +10,19 @@
 			<div class="headline-mask">
 				&nbsp;
 			</div>
-			<div class="headline-main">
-				<div class="headline-background" :style="headlineBgStyles"></div>
-				<div class="headline-title" v-if="hasHeadlineTitle">
-					<slot name="headlineTitle"></slot>
+			<div>
+				<!-- eslint-disable-next-line max-len -->
+				<div class="headline-main" :class="{ 'headline-main--has-action' : hasAction }" v-if="hasHeadlineTitle || hasHeadlineBody">
```

Comment:
You can remove the eslint exception here and drop the attributes to the next line